.github/workflows/code-quality.yml

```diff
@@ -27,34 +25,26 @@ permissions: read-all
 concurrency:
   group: ${{ github.workflow }}-${{ github.event_name }}-${{ contains(github.event_name, 'pull_request') && github.event.pull_request.head.ref || github.sha }}
   cancel-in-progress: true
-env:
-  # set DBT_ADAPTERS_BRANCH to the input value if the event is a workflow_dispatch (workflow_call uses the same event_name),
-  # otherwise use 'main'
-  DBT_ADAPTERS_BRANCH: ${{ github.event_name == 'workflow_dispatch' && inputs.dbt_adapters_branch || 'main' }}
+
 jobs:
   code-quality:
     name: Code Quality
     runs-on: ubuntu-latest
 
-
     steps:
       - name: Check out repository
         uses: actions/checkout@v4
         with:
           persist-credentials: false
 
       - name: Update Adapters and Core branches
+        if: ${{ contains(github.event_name, 'workflow_') }}
         shell: bash
-        run: |
-          ./.github/scripts/update_dev_packages.sh \
-            $DBT_ADAPTERS_BRANCH \
-            "main"
+        run: ./.github/scripts/update_dev_packages.sh ${{ inputs.dbt_adapters_branch }} "main"
 
       - name: Setup `hatch`
-        uses: dbt-labs/dbt-adapters/.github/actions/setup-hatch@main
-
-      - name: Run linters
-        run: hatch run lint:all
+        uses: dbt-labs/dbt-adapters/.github/actions/setup-hatch@config/add-precommit
 
-      - name: Run typechecks
-        run: hatch run typecheck:all
+      - name: Run code quality
+        shell: bash
+        run: hatch run code-quality
```

Comment:
I still don't know what this is 😭

Reply:
https://github.com/dbt-labs/dbt-postgres/blob/805dc3b58826e50a9c33780cbc25aa23331d27a8/pyproject.toml#L73